backend/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.10 AS os-base
+
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="/root/.local/bin:$PATH"
+RUN poetry config virtualenvs.create false
+RUN apt-get remove -y curl
+
+FROM os-base AS app-base
```

Comment:
Not that there's any real issue with introducing another stage here, but does it add anything?

Reply:
Not really, just thought it was nice to have the distinction between the machine config used to run the app and the image with the app itself.